src/style.css

```diff
@@ -0,0 +1,26 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-position: center;
+  flex-direction: row;
```

Comment:
i think you need to move it to `.stars`

Reply:
I try to do this, but it returm an error

Reply:
and i have demo link in the project